.bazelversion

```diff
@@ -1,4 +1,4 @@
-6.0.0rc1
+6.0.0rc2
```

Comment:
I'll pre-factor this one

Reply:
When I downgrade I get the following error with aspect_bazel_lib

```
/private/var/tmp/_bazel_mrmeku/012d6fe4c372246174f93c41cb03f436/external/aspect_bazel_lib/lib/private/docs/BUILD.bazel:10:12: no such target '@local_config_platform//:constraints.bzl': target 'constraints.bzl' not declared in package '' defined by /private/var/tmp/_bazel_mrmeku/012d6fe4c372246174f93c41cb03f436/external/local_config_platform/BUILD.bazel; however, a source file of this name exists.  (Perhaps add 'exports_files(["constraints.bzl"])' to /BUILD?) and referenced by '@aspect_bazel_lib//lib/private/docs:local_config_platform_constraints'
```

Reply:
I'll resolve this since you've run the pre-factor. Thank you!